src/overlays/actors/ovl_En_Fall/z_en_fall.c

```diff
@@ -516,65 +519,65 @@ void EnFall_FireBall_SetPerVertexAlpha(f32 fireBallAlpha) {
 
     s32 pad;
     u8 perVertexAlphaTable[5];
-    Vtx* vertices = Lib_SegmentedToVirtual(&object_fall_Vtx_0004C0);
+    Vtx* vertices = Lib_SegmentedToVirtual(gMoonFireballVtx);
     s32 i;
 
-    if (fireBallAlpha > 1.0f) {
-        fireBallAlpha = 1.0f;
+    if (fireballAlpha > 1.0f) {
+        fireballAlpha = 1.0f;
     }
     perVertexAlphaTable[0] = 0;
-    perVertexAlphaTable[1] = (s8)(255.0f * fireBallAlpha);
-    perVertexAlphaTable[2] = (s8)(155.0f * fireBallAlpha);
-    perVertexAlphaTable[3] = (s8)(104.0f * fireBallAlpha);
-    perVertexAlphaTable[4] = (s8)(54.0f * fireBallAlpha);
+    perVertexAlphaTable[1] = (s8)(255.0f * fireballAlpha);
+    perVertexAlphaTable[2] = (s8)(155.0f * fireballAlpha);
+    perVertexAlphaTable[3] = (s8)(104.0f * fireballAlpha);
+    perVertexAlphaTable[4] = (s8)(54.0f * fireballAlpha);
 
     for (i = 0; i < ARRAY_COUNT(sAlphaTableIndex); i++, vertices++) {
         vertices->v.cn[3] = perVertexAlphaTable[sAlphaTableIndex[i]];
     }
 }
 
-void EnFall_FireBall_Update(Actor* thisx, GlobalContext* globalCtx) {
+void EnFall_Fireball_Update(Actor* thisx, GlobalContext* globalCtx) {
     EnFall* this = THIS;
 
     if (globalCtx->sceneNum == SCENE_00KEIKOKU && gSaveContext.sceneSetupIndex == 0 && globalCtx->csCtx.unk_12 == 2) {
         globalCtx->skyboxCtx.rotY -= 0.05f;
     }
 
     if (func_800EE29C(globalCtx, 0x1C2)) {
-        this->actor.draw = EnFall_FireBall_Draw;
-        if (this->flags & EN_FALL_FLAG_FIRE_BALL_INTENSIFIES) {
-            this->fireBallIntensity += 0.01f;
-            if (this->fireBallIntensity > 1.0f) {
-                this->fireBallIntensity = 1.0f;
+        this->actor.draw = EnFall_Fireball_Draw;
+        if (this->flags & FLAG_FIRE_BALL_INTENSIFIES) {
+            this->fireballIntensity += 0.01f;
+            if (this->fireballIntensity > 1.0f) {
+                this->fireballIntensity = 1.0f;
             }
         }
         func_800EDF24(&this->actor, globalCtx, func_800EE200(globalCtx, 0x1C2));
 
         switch (globalCtx->csCtx.npcActions[func_800EE200(globalCtx, 0x1C2)]->unk0) {
```

Comment:
Here and elsewhere, I think it would be good to have blank lines between the `break` and the next case, to make it a bit easier to read.

If you really want to go all the way, you could note by any defaults which cases they cover, providing it's only a few.